src/amber/router/router.cr

```diff
@@ -75,13 +77,10 @@ module Amber
         end
       end
 
-      def match(http_verb, resource) : Radix::Result(Amber::Route)
+      def match?(http_verb, resource) : Amber::Route?
```

Comment:
this will probably have to change when route parameters are actually handled by the router